dataset:
  lid_topic: /cloud_registered_body
  odom_topic: /Odometry
  stop_distance: 0.1
  down_size: 0.1

groundseparate:
  sensor_height: 1                                # 用于计算开始时的最低高度
  tau_seeds: 0.9
  tau_dis: 0.2

otd:
  tau_ratio: 0.7
  voxel_size: 0.5

